shared/templates/template_OVAL_mount_option_removable_partitions

```diff
@@ -1,39 +1,31 @@
 <def-group>
-  <definition class="compliance" id="mount_option_{{{ MOUNTOPTION }}}_removable_partitions" version="4">
+  <definition class="compliance" id="mount_option_{{{ MOUNTOPTION }}}_removable_partitions" version="5">
     <metadata>
       <title>Add {{{ MOUNTOPTION }}} Option to Removable Media Partitions</title>
       {{{- oval_affected(products) }}}
       <description>The {{{ MOUNTOPTION }}} option should be enabled for all removable devices mounts in /etc/fstab.</description>
     </metadata>
     <criteria operator="OR">
-      <!-- First check if specified removable partition truly exists on the system. If not, don't check /etc/fstab & runtime configuration
+      <!-- First check if specified removable partition truly exists on the system. If not, don't check /etc/fstab
            since there's no device to check against -->
       <extend_definition comment="Check if removable partition really exists on the system"
       definition_ref="removable_partition_doesnt_exist" />
       <!-- Removable device exists. Check if it's CD/DVD drive. If so, verify that at least one from all of the possible its alternative
-           names in /etc/fstab & runtime configuration are configured with '{{{ MOUNTOPTION }}}' option -->
+           names in /etc/fstab are configured with '{{{ MOUNTOPTION }}}' option -->
       <criteria operator="AND">
         <extend_definition comment="Check if removable partition value represents CD/DVD drive"
         definition_ref="var_removable_partition_is_cd_dvd_drive" />
         <criteria operator="OR">
-          <criteria operator="AND">
-            <criterion test_ref="test_{{{ MOUNTOPTION }}}_etc_fstab_cd_dvd_drive"
+          <criterion test_ref="test_{{{ MOUNTOPTION }}}_etc_fstab_cd_dvd_drive"
             comment="Check if at least one from CD/DVD drive alternative names is using '{{{ MOUNTOPTION }}}' mount option in /etc/fstab" />
-            <criterion test_ref="test_{{{ MOUNTOPTION }}}_runtime_cd_dvd_drive"
-            comment="Check if at least one from CD/DVD drive alternative names is using '{{{ MOUNTOPTION }}}' mount option in runtime configuration" />
-          </criteria>
           <extend_definition definition_ref="no_cd_dvd_drive_in_etc_fstab"
           comment="Check if CD/DVD drive is not configured to automount in /etc/fstab" />
         </criteria>
       </criteria>
       <!-- Removable device exists & isn't CD/DVD drive. Check the particular devices is configured with '{{{ MOUNTOPTION }}}' mount option in both
-           /etc/fstab & runtime configuration -->
-      <criteria operator="AND">
-        <criterion test_ref="test_{{{ MOUNTOPTION }}}_etc_fstab_not_cd_dvd_drive"
+           /etc/fstab -->
+      <criterion test_ref="test_{{{ MOUNTOPTION }}}_etc_fstab_not_cd_dvd_drive"
         comment="Check if removable partition is using '{{{ MOUNTOPTION }}}' mount option in /etc/fstab" />
-        <criterion test_ref="test_{{{ MOUNTOPTION }}}_runtime_not_cd_dvd_drive"
```

Comment:
Runtime needs to be kept and doesn't really need to be mentioned in the prose. There is a expectation which is mentioned in the prose to reboot after configuration has happened when a rule fails.

Reply:
Maybe I am missing something - it seems to me that the check won't pass if there is no removable partition mounted (no objects would be found in that case). Removable devices are not supposed to be mounted most of the time, so forcing them to be present during the scan feels inappropriate.

Reply:
I also don't understand the meaning of the runtime check in this OVAL. Would that mean that users are supposed to pass this check only with CD inserted in the drive?

Reply:
I think the idea is that if a removable partition is found, it needs to have the correct mount options, not to force that it is mounted.

Reply:
I agree with @yuumasato